test/trigger_cellranger_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# note, this test must be run from the workflow root for the relative paths to work out.
+# WDL file should be the KCO cellranger, located here:
+# https://portal.firecloud.org/#methods/single-cell-portal/cell-ranger-2-0-2/3/wdl
+
+set -e
+
+# This secrets file contains the following information on a cromwell instance:
+# username
+# password
+# url
```

Comment:
nit: It could be even better to include an example cromwell url here, to clarify whether to include paths like `/v1` in the url for this script.